src/libServer/LookupServer.cpp

```diff
@@ -1143,6 +1147,10 @@ Json::Value LookupServer::TxBlockListing(unsigned int page) {
       m_mediator.m_txBlockChain.GetLastBlock().GetHeader().GetBlockNum();
   Json::Value _json;
 
+  if (currBlockNum == INIT_BLOCK_NUMBER) {
+    throw JsonRpcException(RPC_IN_WARMUP, "No tx blocks");
```

Comment:
Not sure if it should be capitalized.
```suggestion
    throw JsonRpcException(RPC_IN_WARMUP, "No TX blocks");
```

Reply:
Think `Tx` would be better, thanks